.gitignore

```diff
@@ -14,3 +14,4 @@ mono_crash.*
 todo.gd
 .gut_editor_config.json
 *.swp
+venv/
```

Comment:
**tought:** With pre-commit sandboxing the python git hooks, I don't think we need _any_ reference to python in this project. Therefore, we could also remove the `.venv/` ignore from this file.